### PR TITLE
Header comment compacted, fix invalid JSON in header

### PR DESCRIPTION
**Pull request by krk**
Ref https://github.com/quicktype/quicktype/issues/516

### CODE REVIEW

File: src/Language/Rust.ts [hunk 1]
```diff
@@ -332,26 +332,23 @@ class RustRenderer extends ConvenienceRenderer {
     };
 
     protected emitUsageExample(): void {
+        const topLevelName = this.topLevels.keys().next().value;
```
Comment (schani) [inline src/Language/Rust.ts]: If you want to get the first top level name, use `defined(this.topLevels.keySeq().first())`

File: src/Language/Rust.ts [hunk 2]
```diff
@@ -332,26 +332,23 @@ class RustRenderer extends ConvenienceRenderer {
...
 extern crate serde;
-
 #[macro_use]
 extern crate serde_derive;
-
 extern crate serde_json;
 
-use generated_module::TopLevel;
+use generated_module::` +
+                topLevelName +
```
Comment (schani) [inline src/Language/Rust.ts]: You can use string interpolation here, i.e. `${topLevelName}`

**Comment by dvdsgl** [comment]
@krk could the header comments be line comments, each starting with `// ` as we use in other languages? Would this be conventional for Rust?

**Comment by krk** [comment]
Line comments are preferred in API docs according to this RFC: https://github.com/rust-lang/rfcs/blob/master/text/0505-api-comment-conventions.md#use-line-comments

**Pull request merged**